src/main/resources/com/google/api/codegen/nodejs/main.snip

```diff
@@ -139,227 +132,191 @@
+@private bundlingDescriptors(descriptors)
+  var bundleDescriptors = {
+    @join descriptor : descriptors on {@", "}.add(BREAK)
+      {@descriptor.methodName}: new gax.BundleDescriptor(
+        '{@descriptor.bundledFieldName}',
+        [
+          @join fieldName : descriptor.discriminatorFieldNames() on {@", "}.add(BREAK)
+            '{@fieldName}'
+          @end
+        ],
+        @if descriptor.hasSubresponseField()
+          '{@descriptor.subresponseFieldName}',
+        @else
+          null,
         @end
-      @end
+        {@descriptor.byteLengthFunctionName})
+    @end
+  };
+@end
 
-      {@constructDefaults(service)}
+@private longrunningSetup(descriptors)
+  this.operationsClient = new gax.lro({
+    auth: gaxGrpc.auth,
+    grpc: gaxGrpc.grpc
+  }).operationsClient(opts);
 
-      var self = this;
+  this.longrunningDescriptors = {
+    @join descriptor : descriptors on {@", "}.add(BREAK)
+      {@descriptor.methodName}: new gax.LongrunningDescriptor(
+        this.operationsClient,
+        grpcClients.{@descriptor.operationPayloadTypeName}.decode,
+        grpcClients.{@descriptor.metadataTypeName}.decode)
+    @end
+  };
+@end
 
-      this.auth = gaxGrpc.auth;
-      @join stub : stubs on BREAK.add(BREAK)
-        var {@stub.name} = gaxGrpc.createStub(
-            grpcClients.{@stub.fullyQualifiedType},
-            opts);
-        var {@stub.stubMethodsArrayName} = [
-          @join method : {@stub.methodNames} on {@","}.add(BREAK)
-              '{@method}'
-          @end
-        ];
-        {@stub.stubMethodsArrayName}.forEach(function(methodName) {
-          self['_' + methodName] = gax.createApiCall(
-            {@stub.name}.then(function({@stub.name}) {
-              return function() {
-                var args = Array.prototype.slice.call(arguments, 0);
-                return {@stub.name}[methodName].apply({@stub.name}, args);
-              };
-            }),
-            defaults[methodName],
-            {@getDescriptors(service)});
-        });
-      @end
-    }
-  @end
+@private constructDefaults(xapi)
+  var defaults = gaxGrpc.constructSettings(
+      '{@xapi.interfaceKey}',
+      configData,
+      opts.clientConfig,
+      {'x-goog-api-client': googleApiClient.join(' ')});
 @end
 
-@private getDescriptors(service)
-  @if context.validDescriptorsNames(service)
-    @join descriptor : context.validDescriptorsNames(service) on {@" || "}
-      {@descriptor}[methodName]
+@private getDescriptors(xapi)
+  @if xapi.validDescriptorsNames
+    @join name : xapi.validDescriptorsNames() on {@" || "}
+      {@name}[methodName]
     @end
   @else
     null
   @end
 @end
 
-@private pathTemplateSection(service, apiName)
-  @join pathTemplate : context.getPathTemplates(service) on BREAK
+@private pathTemplateSection(xapi)
+  @join pathTemplate : xapi.pathTemplates on BREAK
     var {@pathTemplate.name} = new gax.PathTemplate(
-        '{@pathTemplate.pattern}');
+        '{@pathTemplate.pattern®}');
```

Comment:
Agh! I fat-fingered option when doing a find and replace in intellij causing a trademark sign.